src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java

```diff
@@ -896,14 +896,20 @@ private static void validatePropertySectionPropertyTokens(String fileName, Strin
         final String defaultTokenText = columns.get(3).getTextContent().trim();
         final String expectedDefaultTokenText = CheckUtil.getTokenText(check.getDefaultTokens(),
                 check.getRequiredTokens());
-        assertEquals(expectedDefaultTokenText, defaultTokenText
-                        .replaceAll("\\s+", " ")
-                        .replaceAll("\\s,", ",")
-                        .replaceAll("\\s\\.", "."),
-                fileName + " section '" + sectionName + "' should have all the default tokens");
-        assertFalse(isInvalidTokenPunctuation(defaultTokenText),
-                fileName + "'s default token section: " + sectionName
-                        + "should have ',' & '.' at beginning of the next corresponding lines.");
+        if (expectedDefaultTokenText.isEmpty()) {
+            assertEquals("empty", defaultTokenText,
+                    "Empty tokens should have 'empty' string in xdoc");
+        }
+        else {
+            assertEquals(expectedDefaultTokenText, defaultTokenText
+                            .replaceAll("\\s+", " ")
+                            .replaceAll("\\s,", ",")
+                            .replaceAll("\\s\\.", "."),
+                    fileName + " section '" + sectionName + "' should have all the default tokens");
+            assertFalse(isInvalidTokenPunctuation(defaultTokenText),
+                    fileName + "'s default token section: " + sectionName
+                           + "should have ',' & '.' at beginning of the next corresponding lines.");
```

Comment:
`should have ',' & '.'`
1. Probably 'and' should be 'or'
2. arguably it is better to make it `should have comma and dot .... ` instead of chars

Reply:
> 1.

 fixed.

> 2.

I would keep it exact.